evap/evaluation/templates/base.html

```diff
@@ -181,8 +181,11 @@
 
             // activate clickable hover tables
             document.querySelectorAll(".table-seamless-links [data-url]").forEach(row => {
-                row.addEventListener("click", () => {
-                    window.location.assign(row.dataset.url);
+                row.addEventListener("click", event => {
+                    // We navigate via the <a> tag to enable browser behavior for Ctrl-Click.
```

Comment:
maybe add a short comment why we do this so we don't go back to creating `a` tags. Something like
> HTML disallows `a` as a direct child of `tr`.